Sources/xcprojectlint-package/Validation.swift

```diff
@@ -16,30 +16,34 @@ import Foundation
 import SPMUtility
 
 public enum Validation: String, StringEnumArgument {
-  case buildSettingsExternalized = "build-settings-externalized"
```

Comment:
Another file hit by some weird indentation change.  Did you run our swiftformat?